config/services-uat.yml

```diff
@@ -570,6 +570,50 @@ https://cmr.uat.earthdata.nasa.gov:
           exists: ['shapefileSubset', 'spatialSubset']
       - image: !Env ${HARMONY_METADATA_ANNOTATOR_IMAGE}
 
+  - name: sds/HOSS-HRS-GeoTIFF
+    description: |
+      A service chain for applying the Harmony Metadata Annotator to HOSS/Maskfill
+      output, producing CF compliant annotated NetCDF4 outputs. This service chain
+      is also intended to be used for producing regridded outputs using the harmony-regridder 
+      and  to produce GeoTIFF outputs using net2cog.
+    data_operation_version: '0.21.0'
+    type:
+      <<: *default-turbo-config
+      params:
+        <<: *default-turbo-params
+        env:
+          <<: *default-turbo-env
+          STAGING_PATH: public/sds/HOSS-HRS-GeoTIFF
+    umm_s: S1273752002-EEDTEST
+    capabilities:
+      subsetting:
+        temporal: true
+        bbox: true
+        dimension: true
+        shape: true
+        variable: true
+      reprojection: true
+      output_formats:
+        - application/netcdf # Incorrect mime-type, remove when no longer needed
+        - application/x-netcdf4
+        - image/tiff
+    steps:
+      - image: !Env ${QUERY_CMR_IMAGE}
+        is_sequential: true
+      - image: !Env ${HOSS_IMAGE}
+        operations: ['variableSubset', 'spatialSubset', 'dimensionSubset', 'shapefileSubset', 'temporalSubset']
```

Comment:
Right now you have it configured for both HOSS and maskfill to perform bbox and shapefile subsetting.

I'm not positive but I'm assuming you want to use HOSS if there's a bbox provided, and maskfill if it's a shapefile?

In that case you would want to remove shapefileSubset from HOSS and change maskfill to only specify shapefileSubset for both `operations` and `conditional`.

If maskfill handles both bbox and shapefile subsetting then maybe you want to drop both spatialSubset and shapefileSubset from HOSS and have maskfill perform it?

Reply:
From an operations point of view, HOSS does all the subsetting and MaskFill does something Harmony does not have an operation for.  I'm guessing the HOSS step is OK as is, and the only question is how to attribute the MaskFill step.  We just copied what was in place before for HOSS, so I'd be inclined to leave it that way in the copy.  However, if it is incorrect to specify an operation that doesn't apply to the step, and it is ok to leave the operation off (as none exactly describes what MaskFill does), we could drop operation from the MaskFill step.  Is that the preference?

Reply:
Sounds good and also received some clarification from Owen - I don't think you need to change anything.